tox.ini

```diff
@@ -1,22 +1,24 @@
 [tox]
 envlist =
-    py27,flake8
+    flake8,
+    py27,
+    coverage,
 
 [testenv]
-deps =
-    zope.testrunner
-    zope.app.testing
-    zope.app.wsgi >= 4.0dev
-    zope.app.zcmlfiles
-    zope.app.server
-    zope.testbrowser[test]
-# install zope.app.pagetemplate explicitly to avoid
-# https://github.com/pypa/pip/issues/854
+extras = test
 commands =
-    pip install zope.app.pagetemplate
     zope-testrunner --test-path=src
 
+[testenv:coverage]
+basepython = python2.7
+commands =
+    coverage run -m zope.testrunner --test-path=src {posargs:--auto-color -vv}
+    coverage report --fail-under=92
+extras = test
```

Comment:
I think this is not needed: tox should inherit the `extras` from the default `[testenv]`.

Reply:
You are right.